src/Moryx.Runtime.Endpoints/Modules/Endpoint/ModulesController.cs

```diff
@@ -123,7 +121,7 @@ public ActionResult Reincarnate([FromRoute] string moduleName)
             if (module == null)
                 return NotFound($"Module with name \"{moduleName}\" could not be found");
 
-            _parallelOperations.ExecuteParallel(_moduleManager.ReincarnateModule, module);
+            Task.Run(() => _moduleManager.ReincarnateModuleAsync(module));
```

Comment:
This will silently hide all exceptions thrown by the method, won't it? Should be add a try-catch in the run method with a logger at least, and if possible a different method return 🤔